Lombiq.JsonEditor/ViewModels/EditContentItemViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OrchardCore.ContentManagement;
+
+namespace Lombiq.JsonEditor.ViewModels;
+
+public record EditContentItemViewModel(
```

Comment:
Nice! Perhaps using records for view models should be our general recommendation?

Reply:
This works because it's an MVC view model. For those it's best to always use records. Unfortunately driver view models can't be records, because they have to be `Shape` under the hood and that's a mutable type.